README.md

```diff
@@ -1,48 +1,66 @@
 # JWT
-A [JWT](https://jwt.io/) implementation that uses [nkeys](https://github.com/nats-io/nkeys) to digitally sign JWT tokens. 
-Nkeys use [Ed25519](https://ed25519.cr.yp.to/) to provide authentication of JWT claims.
 
+A [JWT](https://jwt.io/) implementation that uses [nkeys](https://github.com/nats-io/nkeys) to digitally sign JWT tokens.
+Nkeys use [Ed25519](https://ed25519.cr.yp.to/) to provide authentication of JWT claims.
 
 [![License Apache 2](https://img.shields.io/badge/License-Apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 [![ReportCard](http://goreportcard.com/badge/nats-io/jwt)](http://goreportcard.com/report/nats-io/jwt)
 [![Build Status](https://travis-ci.org/nats-io/jwt.svg?branch=master)](http://travis-ci.org/nats-io/jwt)
 [![GoDoc](http://godoc.org/github.com/nats-io/jwt?status.png)](http://godoc.org/github.com/nats-io/jwt)
 [![Coverage Status](https://coveralls.io/repos/github/nats-io/jwt/badge.svg?branch=master&t=NmEFup)](https://coveralls.io/github/nats-io/jwt?branch=master)
 
+## Using the Library
+
+There are several JWT types. Each is associated with a specific NKey type. For example, an Account Claim is associated
+with an Account NKey. Accounts are special in one respect, they can be signed by Account NKeys or Operator Nkeys. When loaded
+into the nats-server the Operator signed version is used, but for a service like NGS the self-signed version can be used to
+prove ownership. The other types are generally only signed by a single NKey type.
```

Comment:
This is specific to signing a request to be sent to operator or account IMO.

Reply:
I will revert the file, and make a release notes